app/src/main/res/layout-land/profile_edit_activity.xml

```diff
@@ -142,8 +142,8 @@
       <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginTop="24dp"
-        android:layout_marginBottom="8dp"
+        android:layout_marginTop="60dp"
```

Comment:
Please add text color to profile_edit_allow_download_sub.

Reply:
The text color code is #00000061 as per the mock. This color very light and isnt showing. Can you help me with this?